lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java

```diff
@@ -23,11 +27,11 @@ public interface ClassificationService {
    *     Classification}
    * @param domain the {@linkplain Classification#getDomain() domain} of the searched-for
    *     {@linkplain Classification}
-   * @return if exist: domain-specific {@linkplain Classification}, else master {@linkplain
+   * @return if exists: domain-specific {@linkplain Classification}, else master {@linkplain
    *     Classification}
-   * @throws ClassificationNotFoundException if no {@linkplain Classification} is found that matches
-   *     the {@linkplain Classification#getKey() key} either in {@linkplain
-   *     Classification#getDomain() domain} or in the master {@linkplain Classification#getDomain()
+   * @throws ClassificationNotFoundException if no {@linkplain Classification} with specified
+   *     {@linkplain Classification#getKey() key} was found neither in the specified {@linkplain
+   *     Classification#getDomain() domain} nor in the master {@linkplain Classification#getDomain()
```

Comment:
not your fault, but master domain is not an attribute. It's a specific value of `getDomain()`.
Idea: Let's create a constant which we call `MASTER_DOMAIN` and reference that. 
Additionally, this can clear up some of our code.
=> can you create a ticket for this please? A subticket for the documentation ticket should be enough. 
Before starting with that ticket we should discuss it briefly in one of our backlog groomings.

Reply:
Ok :) Can I create two tickets? One subticket for documentation, and one separate ticket? The clean up would affect both, the code and the documentation

Reply:
sure thing. Organize the tickets however you're feeling fine :)

Reply:
Please resolve this conversation as soon as you've created the tickets ;)